### PR TITLE
Hard stop worker by sending ctrl-c twice

### PR DESCRIPTION
**Pull request by mgax**
Fixes https://github.com/RealOrangeOne/django-tasks/issues/84.

### CODE REVIEW

File: django_tasks/backends/database/management/commands/db_worker.py [hunk 1]
```diff
@@ -33,16 +34,22 @@ def __init__(
...
+            signal.strsignal(signum),
         )
-
         self.running = False
 
         if not self.running_task:
             # If we're not currently running a task, exit immediately.
             # This is useful if we're currently in a `sleep`.
-            exit(0)
+            sys.exit(0)
```
Comment (mgax) [inline django_tasks/backends/database/management/commands/db_worker.py:52]: `exit` is meant for the console; `sys.exit` is meant for code.

Source: https://stackoverflow.com/a/6501134/51801

File: django_tasks/backends/database/management/commands/db_worker.py [hunk 2]
```diff
@@ -94,8 +101,9 @@ def start(self) -> None:
                 # If we're running in "batch" mode, terminate the loop (and thus the worker)
                 return None
 
-            # Wait before checking for another task
-            time.sleep(self.interval)
+            if self.running:
+                # Wait before checking for another task
+                time.sleep(self.interval)
```
Comment (mgax) [inline django_tasks/backends/database/management/commands/db_worker.py:106]: If ctrl-c has just interrupted a task, we should not sleep, but rather exit immediately.

File: django_tasks/backends/database/management/commands/db_worker.py (hunk 1 above)
Comment (RealOrangeOne) [reply django_tasks/backends/database/management/commands/db_worker.py:52]: TIL!

File: django_tasks/backends/database/management/commands/db_worker.py (hunk 2 above)
Comment (RealOrangeOne) [reply django_tasks/backends/database/management/commands/db_worker.py:106]: Make sense to me - nice catch! Could that be immortalised as a comment?

**Review by RealOrangeOne — APPROVED** [review]
A nit for a comment, but otherwise this looks great.

File: django_tasks/backends/database/management/commands/db_worker.py (hunk 2 above)
Comment (mgax) [reply django_tasks/backends/database/management/commands/db_worker.py:108]: I thought you might ask that :) I've added the comment.

**Pull request merged**